src/components/inputAndLabel/InputField.tsx

```diff
@@ -1,30 +1,23 @@
 import { InputFieldStyled } from "./InputAndLabel.styles.ts";
 import { InputFieldProps } from "./InputAndLabel.types.ts";
+import { forwardRef } from "react";
```

Comment:
put this page in your brower's boomarks, pls :)  https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/forward_and_create_ref/

Once you figure this out, there will be one minor issue left with naming the export. We can deal with it later or you can peek into our library and see how it's done.